src/main/java/core/basesyntax/dao/impl/AbstractDao.java

```diff
@@ -1,11 +1,93 @@
 package core.basesyntax.dao.impl;
 
+import core.basesyntax.dao.GenericDao;
+import java.util.List;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.hibernate.query.Query;
 
-public abstract class AbstractDao {
+public abstract class AbstractDao<T> implements GenericDao<T> {
```

Comment:
let's change it according to Roman comments in the Slack))

Reply:
Going to do that)